ios-competency-test/ios-competency-test/ViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ViewController.swift
+//  ios-competency-test
+//
+//  Created by Sam Gubler on 3/27/15.
+//  Copyright (c) 2015 Sam Gubler. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    var friends : [Friend] = []
+
+    @IBOutlet weak var tableView: UITableView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        self.friends = FriendCommand.getListOfFriends()
```

Comment:
You're right this does.  I figured with the app having nothing to show until the the call returned I would just let it hang.  I get that it's a poor solution in production code as it is possible the service is not available.  Had I done it the other way I would have done it as we discussed (Have ViewController be a delegate for FriendCommand, then signal back when call completes).